py-polars/polars/functions/lazy.py

```diff
@@ -127,26 +149,76 @@ def count(column: str | None = None) -> Expr:
     ╞═════╡
     │ 2   │
     └─────┘
+
+    Return the number of non-null values in multiple columns.
+
+    >>> df.select(pl.count("b", "c"))
+    shape: (1, 2)
+    ┌─────┬─────┐
+    │ b   ┆ c   │
+    │ --- ┆ --- │
+    │ u32 ┆ u32 │
+    ╞═════╪═════╡
+    │ 1   ┆ 3   │
+    └─────┴─────┘
+
     """  # noqa: W505
-    if column is None:
+    if columns is None:
```

Comment:
If we go for a simple `*columns: str` signature, this can become a simple `if not columns`.